tests-scan

```diff
@@ -287,6 +287,21 @@ def cockpit_tasks(api, update, contexts, repo, pull_data, pull_number, sha, amqp
         else:
             logging.error("Can't find pull request %s", pull_number)
             return 1
+    elif sha:
+        pulls.append({
+            "title": f"{sha}",
+            "number": 0,
+            "head": {
+                "sha": sha,
+                "user": {
+                    "login": "cockpit-project"
+                }
+            },
+            "base": {
+                "ref": "main"  # TODO: anaconda/cockpit-certificates
```

Comment:
perhaps change the comment to "projects which have a "master" branch, to clarify what it means? But this is actually easy to do, testmap.py has a `get_default_branch(repo)` function.